include/unifex/detail/concept_macros.hpp

```diff
@@ -387,6 +387,13 @@ UNIFEX_CONCEPT  //
     callable =  //
     sizeof(decltype(_is_callable::_try_call(static_cast<Fn (*)(As...)>(
         nullptr)))) == sizeof(_is_callable::yes_type);
+
+template <typename Ret, typename... As>
+UNIFEX_CONCEPT              //
+    callable<Ret(As...)> =  //
+    sizeof(decltype(_is_callable::_try_call(
+        static_cast<std::add_pointer_t<Ret(As...)> (*)(As...)>(nullptr)))) ==
+    sizeof(_is_callable::yes_type);
```

Comment:
What's this for? I _think_ the explicit specialization is misaligned with the intended usage.